src/tools/sdk/_windows.md

```diff
@@ -1,23 +1,41 @@
+###How to install?
+
 You can install the Dart SDK using [Chocolatey.][Chocolatey]
 
-{{site.alert.important}}
-These commands require administrator privileges.
-If you need help on starting an administrator-level command prompt,
-try a search like
-<em><a href="https://www.google.com/search?q=cmd+admin"
-  target="blank">cmd admin</a>.</em>
-{{site.alert.end}}
+> **_NOTE:_** These commands require administrative privileges. To start command prompt as an administrator following these steps:
+> 
+> - Press Windows+R to open the “Run” box. 
+> - Type “cmd” into the box 
+> - Press Ctrl+Shift+Enter to run the command as an administrator.
```

Comment:
For formatting keyboard keys and combinations, use `<kbd>` or backticks (`) as specified in https://developers.google.com/style/ui-elements?hl=en#keyboard

Reply:
Ok sure